modules/ROOT/pages/deployment/services/s-list/policies.adoc

```diff
@@ -165,6 +165,30 @@ Using git version name: `{compose_tab_3}`
 endif::[]
 ====
 
+== Extend Mimetype File Extension Mapping
```

Comment:
Nowadays they are actually called media type... Should we change to that?

Reply:
ok for me